src/main/resources/application.properties

```diff
@@ -2,13 +2,13 @@ spring.application.name=KafkaStudy
 kafka.regular.topic=my-topic
 regular.kafka.autostart=false
 reactive.kafka.autostart=true
-spring.kafka.consumer.bootstrap-servers=127.0.0.1:9092
+spring.kafka.consumer.bootstrap-servers=kafka:9092
```

Comment:
Able to produce message to Kafka and App is consuming data
![image](https://github.com/user-attachments/assets/d9f3c627-54e1-4e9d-8743-c5afb6b92e19)

App logs:
![image](https://github.com/user-attachments/assets/169f2118-c3c4-4094-afe1-f0876c1e0be9)